tests/functional/utils/chart_certification.py

```diff
@@ -348,8 +357,11 @@ def update_test_report(self, test_report):
             self.__post_init__()
 
     def get_unique_vendor(self, vendor):
+        """Set unique vendor name.
+        Note that release tag is generated with this vendor name.
+        """
         # unique string based on current time in seconds
-        suffix = str(int(time.time()))
```

Comment:
what is wrong with time guid is big string, are we going to have collisions within milliseconds. Update the comment if you go with guid

Reply:
For parallel runs the timestamp might be identical, even with time_ns() (nanoseconds). The first batch of tests (the first job from each thread) have a higher chance of collision, I noticed the collision even using the nanosecond timestamp, and after switching to uuid I don't see collisions. Therefore I used uuid, great question.

updating comments to explain this change.

Reply:
OK, thanks for explanation